### PR TITLE
[Window365] add new property alternateResourceUrl in ANC payload

### PR DESCRIPTION
**Pull request by Sodaine**
Roadmap workitem number: 14200

### CODE REVIEW

**Comment by AshleyYangSZ** [comment]
@Sodaine Please include change-log in the PR.

File: api-reference/beta/resources/cloudpconpremisesconnection.md [hunk 1]
```diff
@@ -128,6 +129,7 @@ The following is a JSON representation of the resource.
       }
     ]
   },
-  "inUse": "Boolean"
+  "inUse": "Boolean",
+  "alternateResourceUrl": "string"
```
Comment (AshleyYangSZ) [inline api-reference/beta/resources/cloudpconpremisesconnection.md:133]: If this property returned by default, then you'd better add it to the related API examples.

File: api-reference/beta/resources/cloudpconpremisesconnection.md [hunk 2]
```diff
@@ -49,6 +49,7 @@ Represents a defined collection of Azure resource information that can be used t
...
 |healthCheckStatusDetails|[cloudPcOnPremisesConnectionStatusDetails](../resources/cloudpconpremisesconnectionstatusdetails.md)|The details of the connection's health checks and the corresponding results. Returned only on `$select`.For an example that shows how to get the **inUse** property, see [Example 2: Get the selected properties of an Azure network connection, including healthCheckStatusDetails](../api/cloudpconpremisesconnection-get.md). Read-only.|
 |inUse|Boolean|When `true`, the Azure network connection is in use. When `false`, the connection is not in use. You cannot delete a connection that’s in use. Returned only on `$select`. For an example that shows how to get the **inUse** property, see [Example 2: Get the selected properties of an Azure network connection, including healthCheckStatusDetails](../api/cloudpconpremisesconnection-get.md). Read-only.|
+|alternateResourceUrl|String|The interface URL of the partner service's resource that links to this on-premises connection. Returned only on `$select`.
```
Comment (RuiHou105) [inline api-reference/beta/resources/cloudpconpremisesconnection.md:52]: I remember we already rename on-premises connection to Azure network connection right? So maybe here you cannot use the old naming on-premises. Please help update. Thanks!

File: api-reference/beta/resources/cloudpconpremisesconnection.md (hunk 1 above)
Comment (RuiHou105) [reply api-reference/beta/resources/cloudpconpremisesconnection.md:133]: In the description, this property will only be returned with $select.

**Review by RuiHou105 — CHANGES_REQUESTED** [review]
@Sodaine  please help resolve comments in this PR. Thanks!

File: api-reference/beta/resources/cloudpconpremisesconnection.md (hunk 2 above)
Comment (Sodaine) [reply api-reference/beta/resources/cloudpconpremisesconnection.md]: Thanks for catching it, updated!

**Review by RuiHou105 — APPROVED** [review]

**Comment by RuiHou105** [comment]
@Lauragra  @JarbasHorst , could you help review this PR when you got time? Thanks ahead :)

File: api-reference/beta/resources/cloudpconpremisesconnection.md [hunk 3]
```diff
@@ -49,6 +49,7 @@ Represents a defined collection of Azure resource information that can be used t
...
 |healthCheckStatusDetails|[cloudPcOnPremisesConnectionStatusDetails](../resources/cloudpconpremisesconnectionstatusdetails.md)|The details of the connection's health checks and the corresponding results. Returned only on `$select`.For an example that shows how to get the **inUse** property, see [Example 2: Get the selected properties of an Azure network connection, including healthCheckStatusDetails](../api/cloudpconpremisesconnection-get.md). Read-only.|
 |inUse|Boolean|When `true`, the Azure network connection is in use. When `false`, the connection is not in use. You cannot delete a connection that’s in use. Returned only on `$select`. For an example that shows how to get the **inUse** property, see [Example 2: Get the selected properties of an Azure network connection, including healthCheckStatusDetails](../api/cloudpconpremisesconnection-get.md). Read-only.|
+|alternateResourceUrl|String|The interface URL of the partner service's resource that links to this Azure network connection. Returned only on `$select`.
```
Comment (Lauragra) [inline api-reference/beta/resources/cloudpconpremisesconnection.md:52]: FYI, we list the properties in alphabetical order in the properties tables and in the JSON representation. Please alphabetize the properties here, and also be sure to add the missing | to the end of this table row. Currently the table is scrolling on the review site and I'm not sure why but let's see if that's the fix.

File: api-reference/beta/resources/cloudpconpremisesconnection.md (hunk 1 above)
Comment (Lauragra) [inline api-reference/beta/resources/cloudpconpremisesconnection.md:133]: Alphabetize the properties here, too.

File: api-reference/beta/resources/cloudpconpremisesconnection.md (hunk 3 above)
Comment (Sodaine) [reply api-reference/beta/resources/cloudpconpremisesconnection.md]: updated

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review. Please, merge PR [17133](https://github.com/microsoftgraph/microsoft-graph-docs/pull/17133) into this branch to include my final edits.

**Review by Lauragra — APPROVED** [review]

**Comment by RuiHou105** [comment]
Hi @Lauragra  as the graph schema went alive last week, now let's merge this PR. Thanks !

**Pull request merged**